apps/communications/pbap/js/call_log.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+/* global applications, BrowserConfigHelper, AppWindow */
+/* jshint nonew: false */
+
+(function(exports) {
+  function PbapCallLog() {
+    this.preHandleEvent = this.preHandleEvent.bind(this);
```

Comment:
This isn't doing anything, why do you need this? If there's no specific need then this file must be removed.